core-web/apps/dotcms-ui/src/app/view/components/dot-api-link/dot-api-link.component.scss

```diff
@@ -1,35 +1,25 @@
 @use "variables" as *;
 
-:host {
-    display: inline-flex;
-}
-
 a,
 a:focus,
 a:active {
     text-decoration: none;
 }
 
-.p-button {
-    gap: 0.25rem;
-}
-
-.material-icons {
-    color: $color-palette-primary;
-    font-size: $font-size-default;
-}
-
-// This styles are the same as .p-button.p-button-outlined:enabled:hover in _buttons.scss,
+// This styles are the same as .p-button.p-button-text in _buttons.scss,
```

Comment:
In the future we might want to turn this component into a "link icon" component.